src/main.rs

```diff
@@ -377,6 +384,8 @@ fn check_for_emcc( use_system_emscripten: bool ) -> Option< PathBuf > {
         println_err!( "  sudo apt-get install emscripten" );
     } else if cfg!( target_os = "linux" ) {
         println_err!( "You can most likely find it in your distro's repositories." );
+    } else if cfg!( target_os = "windows" ) {
+        println_err!( "Download and install emscripted from the official site: http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html")
```

Comment:
Typo? `emscripted` -> `emscripten`

Also missing space before `)`. (: (I'll have to set up `rustfmt` sooner or later.)